frontend/src/bundles/common/components/badge/badge.tsx

```diff
@@ -0,0 +1,89 @@
+import { Grid } from '@mui/material';
+import { type ReactElement } from 'react';
+
+import { BadgeColors } from '../../enums/enums.js';
+import { getValidClassNames } from '../../helpers/helpers.js';
+import { BadgeIcon } from '../badge-icon/badge-icon.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+    primaryText: string;
+    description: string;
+    secondText?: string;
+    color?: keyof typeof BadgeColors;
```

Comment:
Can you use ValueOf ?

Reply:
thanks, now it's working as expected `color={BadgeColors.ORANGE}`